test/hd-test.js

```diff
@@ -103,7 +103,7 @@ describe('HD', function() {
   });
 
   it('should inspect Mnemonic', () => {
-    const mne = new HD.Mnemonic();
+    const mne = new HD.Mnemonic({bits: 128});
```

Comment:
Should there also be a test to check that without options the default Mnemonic is 24 words?